### PR TITLE
Add Support for Sunxi Linux Boards

### PR DESCRIPTION
**Pull request by fabys77**
I've added support for boards that use Sunxi Linux like some Olimex and Cubieboard.
I've tested only GPIO on Cubieboard2 (armbian 5.8)

### CODE REVIEW

File: pi_mqtt_gpio/modules/bme280.py [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+
+    def get_value(self, config):
+        """get the temperature or humidity value from the sensor"""
+        print("READING BME")
+        temperature, preassure, humidity = self.readBME280All()
+        if config["type"] == "temperature" and temperature is not None:
+            return temperature
+        if config["type"] == "humidity" and humidity is not None:
+            return humidity
+        if config["type"] == "preassure" and preassure is not None:
```
Comment (flyte) [inline pi_mqtt_gpio/modules/bme280.py]: Typo here, should be `pressure`

File: pi_mqtt_gpio/modules/bme280.py [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+        required=False,
+        empty=False,
+        default="temperature",
+        allowed=["temperature", "humidity", "preassure"],
+    ),
+}
+
+class Sensor(GenericSensor):
+    """
+    Implementation of Sensor class for the LM75 temperature sensor.
```
Comment (flyte) [inline pi_mqtt_gpio/modules/bme280.py]: Docstring needs updating to be relevant to the 'BME280' module

**Review by flyte — DISMISSED** [review]
Thanks for the module! The Sunxi stuff all looks good to me.

However, I'm apprehensive about merging the BME280 module, since it seems to me like code that should exist in its own external package specifically for the BME280 as opposed to something the `pi-mqtt-gpio` project should be supporting. Having done a quick search, would it be possible to use a library such as https://github.com/pimoroni/bme280-python instead?

File: pi_mqtt_gpio/modules/bme280.py (hunk 1 above)
Comment (fabys77) [reply pi_mqtt_gpio/modules/bme280.py]: Resolved with commit b7e75f044c7eb475d6ea36f648c60d8c37c73bf1

File: pi_mqtt_gpio/modules/bme280.py (hunk 2 above)
Comment (fabys77) [reply pi_mqtt_gpio/modules/bme280.py]: correct with b7e75f044c7eb475d6ea36f648c60d8c37c73bf1

**Comment by fabys77** [comment]
Hi flyte. I've followed your advise and and I change bme280 module using this library: 
https://pypi.org/project/RPi.bme280/
Previous implementation was very buggy.
Regards.

Fabio

**Pull request merged**